### ISSUE TITLE
Empty core.min.js produces a lot of errors

### ISSUE THREAD

**Issue by adam-lebon** [CTFd/CTFd#2257]
**Environment**:

- CTFd Version/Commit: CTFd 2.5.1 commit 472010f
- Operating System: Linux (official dockerfile)
- Web Browser and Version: Chrome 109

**What happened?**
With a fresh installation of CTFd, gunicorn emit a lot of errors. It seems that gunicorn doesn't like to serve empty files. This is due to the core.min.js being empty.

**How to reproduce your issue**
This issue happens with any standard CTFD setup since it seems to be caused by the [core.min.js](https://github.com/CTFd/CTFd/blob/master/CTFd/themes/core/static/js/core.min.js) file being empty.

**Any associated stack traces or error logs**
```
[2023-02-12 16:39:34 +0000] [19] [ERROR] Error handling request
Traceback (most recent call last):
  File "/opt/venv/lib/python3.9/site-packages/gunicorn/workers/base_async.py", line 113, in handle_request
    resp.write_file(respiter)
  File "/opt/venv/lib/python3.9/site-packages/gunicorn/http/wsgi.py", line 387, in write_file
    if not self.sendfile(respiter):
  File "/opt/venv/lib/python3.9/site-packages/gunicorn/http/wsgi.py", line 377, in sendfile
    self.sock.sendfile(respiter.filelike, count=nbytes)
  File "/opt/venv/lib/python3.9/site-packages/gevent/_socket3.py", line 495, in sendfile
    return self._sendfile_use_send(file, offset, count)
  File "/opt/venv/lib/python3.9/site-packages/gevent/_socket3.py", line 425, in _sendfile_use_send
    self._check_sendfile_params(file, offset, count)
  File "/opt/venv/lib/python3.9/site-packages/gevent/_socket3.py", line 470, in _check_sendfile_params
    raise ValueError(
ValueError: count must be a positive integer (got 0)
```

**Fix**
Filling the core.min.js file with at least one byte removes the exception.
![image](https://user-images.githubusercontent.com/3957211/218325233-c92de32d-8c51-4116-8912-8ce6e2e5f62e.png)

**Comment by sxmimran** [comment]
Prior to the fix introduced in gunicorn version > 20.1.0, Added a webpack tweak to address the issue. The tweak involves checking if the file is empty and adding a comment, as suggested by @msterhuj. This ensures that we don't lose the comment when regenerating the theme.

Added Fix for empty core.min.js file error: #2365

**Comment by adam-lebon** [comment]
Not relevant anymore, closing
